src/mongocxx/test/collection_mocked.cpp

```diff
@@ -106,6 +107,12 @@ TEST_CASE("Collection", "[collection]") {
                             expected_batch_size);
                 }
 
+                if (opts.bypass_document_validation())
+                    REQUIRE(o["bypassDocumentValidation"].get_bool().value ==
+                            expected_bypass_document_validation);
+                else
+                    REQUIRE(o.find("bypassDocumentValidation") == o.end());
```

Comment:
@samantharitter 's comment went away, not sure why:
> or just `REQUIRE(!o["bypassDocumentValidation"])`

Should I change it to `!o["key"]` for all the prior tests?